.github/workflows/sycl-cts-on-spirv-be.yml

```diff
@@ -0,0 +1,72 @@
+# This workflow builds SYCL-CTS with -fsycl-use-spirv-backend-for-spirv-gen and
+# runs it with opencl:cpu & gen12 and level_zero:gpu & PVC on Sundays.
+
+name: SYCL-CTS with SPIR-V backend
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * 0'
+
+permissions: read-all
+
+jobs:
+  ubuntu2204_build:
+    if: github.repository == 'intel/llvm'
+    uses: ./.github/workflows/sycl-linux-build.yml
+    secrets: inherit
+    with:
+      build_cache_root: "/__w/"
+      build_artifact_suffix: default
+      build_configure_extra_args: ''
+      merge_ref: ''
+
+  build-sycl-cts:
+    needs: ubuntu2204_build
+    if: ${{ always() && !cancelled() && needs.ubuntu2204_build.outputs.build_conclusion == 'success' }}
+    uses: ./.github/workflows/sycl-linux-run-tests.yml
+    with:
+      name: Build SYCL-CTS
+      runner: '["Linux", "build"]'
+      cts_testing_mode: 'build-only'
+      image: ghcr.io/intel/llvm/ubuntu2404_intel_drivers:latest
+      image_options: -u 1001 --device=/dev/dri --privileged --cap-add SYS_ADMIN
+      tests_selector: cts
+      ref: ${{ github.sha }}
+      sycl_toolchain_artifact: sycl_linux_default
```

Comment:
do we need to set `merge_ref` to empty anywhere in this workflow?

Reply:
Nope, it's only needed for build